index.bs

```diff
@@ -2766,6 +2829,55 @@ The steps for [=issuing a credential request to an authenticator=] are as follow
 
     1. Return [TRUE].
 
+#### Get Request Exceptions #### {#sctn-get-request-exceptions}
+
+[INFORMATIVE]
+
+[=[WRPS]=] can encounter a number of exceptions from a call to {{CredentialsContainer/get()|navigator.credentials.get()}}.
+Some exceptions can have multiple reasons for why they happened,
+requiring the [=[WRPS]=] to infer the actual reason based on their use of WebAuthn.
+
+Note: Exceptions that can be raised during processing of any [=WebAuthn Extensions=],
+including ones defined outside of this specification,
+are not listed here.
+
+The following {{DOMException}} exceptions can be raised:
+
+<dl>
+    :   {{AbortError}}
+    ::  The ceremony was cancelled by an {{AbortController}}.
+        See [[#sctn-abortoperation]] and [[#sctn-sample-aborting]].
+
+    :   {{SecurityError}}
+    ::  The [=effective domain=] was not a [=valid domain=],
+        or <code>{{PublicKeyCredentialCreationOptions/rp}}.{{PublicKeyCredentialRpEntity/id}}</code> was not equal to or a registrable domain suffix of the [=effective domain=].
+        In the case of [[#sctn-related-origins|related origin requests]],
+        the [=authenticator=] does not support it
+        or there were issues when [[#sctn-validating-relation-origin|validating related origins]].
+
+    :   {{TimeoutError}}
+    ::  The ceremony was cancelled by the user agent after exceeding the time limit permitted for the ceremony.
+        See [[#sctn-timeout-recommended-range]] for more information.
+
+    :   {{UnknownError}}
+    ::  The [=authenticator=] could not process the supplied options,
+        or encountered an error while generating an [=assertion signature=].
```

Comment:
I'm not sure UnknownError is actually surfaced to the caller - the only occurrences in the spec are that the authenticator should "return an error code equivalent to UnknownError", but these errors are silently ignored by the 'If any authenticator returns an error status not equivalent to "[InvalidStateError](https://webidl.spec.whatwg.org/#invalidstateerror)"' case in `create()` and the "If any authenticator returns an error status" case in `get()`.

Reply:
As mentioned above, you're right, the RP will get a NotAllowedError from the client after an authenticator raises UnknownError and the user cancels out of the browser modal. I've removed `UnknownError`